render.go

```diff
@@ -0,0 +1,87 @@
+package render
+
+import (
+	"image/color"
+	"image/draw"
+	"math"
+
+	"github.com/benoitkugler/textlayout/fonts"
+	"github.com/go-text/typesetting/shaping"
+	"github.com/srwiley/rasterx"
+	"golang.org/x/image/math/fixed"
+)
+
+type Renderer struct {
+	FontSize, PixScale float32
+	Color              color.Color
+}
+
+func (r *Renderer) DrawString(str string, img draw.Image, face fonts.Face) int {
```

Comment:
Could you document where the first glyph is drawn on `img` ?

Reply:
Thanks for that, fixed